### PR TITLE
Update security-identifiers.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10111
added two rows to Identifier Authority table, removed S-1-16 entries from the list of well-known SIDs
source:
https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/81d92bba-d22b-4a8c-908a-554ab29148ab
https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/c6ce4275-3d90-4890-ab3a-514745e4637e

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/security/identity-protection/access-control/security-identifiers.md [hunk 1]
```diff
@@ -166,14 +166,16 @@ The following table lists the universal well-known SIDs.
 | S-1-5 | NT Authority | A SID that represents an identifier authority. |
 | S-1-5-80-0 | All Services | A group that includes all service processes configured on the system. Membership is controlled by the operating system.|
 
-The following table lists the predefined identifier authority constants. The first four values are used with universal well-known SIDs, and the last value is used with well-known SIDs in Windows operating systems designated in the **Applies To** list.
+The following table lists the predefined identifier authority constants. The first four values are used with universal well-known SIDs, and the rest values are used with well-known SIDs in Windows operating systems designated in the **Applies To** list.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/security-identifiers.md:169]:
```suggestion
The following table lists the predefined identifier authority constants. The first four values are used with universal well-known SIDs, and the rest of the values are used with well-known SIDs in Windows operating systems designated in the **Applies To** list.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MaratMussabekov, kindly commit the minor suggested change. Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you @MaratMussabekov @JohanFreelancer9

**Pull request merged**